1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,22 @@
-function readingList(books) {
-  // Write your code here...
+  function readingList(books) {
```

Comment:
Nicely done! 👍 
I see that You are trying to code in a DRY way. 
https://codinglead.co/javascript/what-is-DRY-code#:~:text=There's%20a%20principle%20in%20programming,any%20change%20in%20one%20place.